tests/conftest.py

```diff
@@ -0,0 +1,94 @@
+import pytest
+import os
+from astrodbkit2.astrodb import create_database, Database
+from astrodb_scripts import load_astrodb
+from simple import REFERENCE_TABLES
+from simple.schema import *
+
+# Create a fresh temporary SIMPLE database and assert it exists
+# Because we've imported simple.schema, we will be using that schema for the database
+# This is in the conftest.py file so temp database is only created once per test session
+
+DB_NAME = "tests/simple_tests.sqlite"
+DB_PATH = "data"
+
+
+if os.path.exists(DB_NAME):
+    os.remove(DB_NAME)
+connection_string = "sqlite:///" + DB_NAME
+create_database(connection_string)
+assert os.path.exists(DB_NAME)
+
+# Connect to the new database and confirm it has the Sources table
+db = Database(connection_string, reference_tables=REFERENCE_TABLES)
+assert db
+assert "source" in [c.name for c in db.Sources.columns]
+
+# Load data into an in-memory sqlite database first, for performance
+temp_db = Database(
+    "sqlite://", reference_tables=REFERENCE_TABLES
+)  # creates and connects to a temporary in-memory database
+temp_db.load_database(
+    DB_PATH, verbose=False
+)  # loads the data from the data files into the database
+temp_db.dump_sqlite(DB_NAME)  # dump in-memory database to file
+db = Database(
+    "sqlite:///" + DB_NAME, reference_tables=REFERENCE_TABLES
+)  # replace database object with new file version
+
+
+@pytest.fixture(scope="session", autouse=True)
+def db():
```

Comment:
In my opinion, all of the logic above should be moved inside the fixture since it's all meant to be supporting the creation of the database. Otherwise you have a bit of messy code where you have two instances of db, one inside the fixture and another inside conftest...

Reply:
My rationale is that all the stuff outside of the function only needs to happen once, at the beginning of the test run.  Like to "initialize" the test suite. I wasn't sure if I put it all in the function, if it would re-do all those things every time `db` was used.

Reply:
I used `pytest -s` to confirm that it only runs these functions once!  Code moved into functions. :)